androidapp/DansFrappesRaps/app/src/main/java/com/cs3450/dansfrappesraps/ui/navigation/RootNavigation.kt

```diff
@@ -51,18 +52,17 @@ fun RootNavigation() {
     ){
         NavHost(
             navController = navController,
-            startDestination = Routes.foyer.route,
+            startDestination = Routes.signIn.route,
             modifier = Modifier.padding(paddingValues = it)
         ) {
-            navigation(route = Routes.foyer.route, startDestination = Routes.signIn.route){
-                composable(route = Routes.signIn.route) { SignInScreen(navHostController = navController)}
-                composable(route = Routes.signUp.route) { SignUpScreen(navController)}
-            }
-            navigation(route = Routes.app.route, startDestination = Routes.menu.route) {
-                composable(route = Routes.menu.route) {MenuScreen(navHostController = navController)}
-            }
-//            composable(route = Routes.signIn.route) { SignInScreen(navHostController = navController)}
-//            composable(route = Routes.menu.route) {MenuScreen(navController)}
+//            navigation(route = Routes.foyer.route, startDestination = Routes.signIn.route){
+//                composable(route = Routes.signIn.route) { SignInScreen(navHostController = navController)}
+//            }
+//            navigation(route = Routes.app.route, startDestination = Routes.menu.route) {
+//                composable(route = Routes.menu.route) {MenuScreen(navHostController = navController)}
+//            }
+            composable(route = Routes.signIn.route) { SignInScreen(navHostController = navController)}
+            composable(route = Routes.menu.route) {MenuScreen(navController)}
```

Comment:
These two composable can be deleted, they were mainly for testing purposes and we are moving to nav graphs from here